frontend/workflows/projectSelector/src/project-links.tsx

```diff
@@ -0,0 +1,168 @@
+import React from "react";
+import type { clutch as IClutch } from "@clutch-sh/api";
+import { Link, Popper, Typography } from "@clutch-sh/core";
+import styled from "@emotion/styled";
+import IconButton from "@material-ui/core/IconButton";
+import MoreVertIcon from "@material-ui/icons/MoreVert";
+
+interface LinkGroupProps {
+  linkGroupName: string;
+  linkGroupImage: string;
+}
+
+interface QuickLinkProps extends LinkGroupProps {
+  link: IClutch.core.project.v1.ILink;
+}
+
+const ICON_SIZE = "22px";
+
+const StyledMenuItem = styled.div({
+  display: "flex",
+  alignItems: "center",
+  "&:hover": {
+    backgroundColor: "rgba(13, 16, 48, 0.05)",
+  },
+});
+
+const StyledMoreVertIcon = styled.span({
+  ".MuiIconButton-root": {
+    padding: "6px",
+    color: "rgba(13, 16, 48, 0.38)",
+  },
+  ".MuiIconButton-root:hover": {
+    backgroundColor: "rgb(245, 246, 253)",
+  },
+  ".MuiIconButton-root:active": {
+    backgroundColor: "rgba(0,0,0, 0.1)",
+  },
+});
+
+const StyledLinkTitle = styled.span({
+  fontWeight: "bold",
+});
+
+const QuickLink = ({ link, linkGroupName, linkGroupImage }: QuickLinkProps) =>
+  link?.url ? (
+    <StyledMenuItem style={{ padding: "8px" }} key={link.url}>
+      <Link href={link.url}>
+        <img
+          width={ICON_SIZE}
+          height={ICON_SIZE}
+          src={linkGroupImage}
+          alt={link.name ?? `Quick Link to ${link.url}`}
+        />
+        <StyledLinkTitle style={{ paddingLeft: "6px" }}>{linkGroupName}</StyledLinkTitle>
+      </Link>
+    </StyledMenuItem>
+  ) : null;
+
+interface QuickLinkGroupProps extends LinkGroupProps {
+  links: IClutch.core.project.v1.ILink[];
+}
+
+const QuickLinkGroup = ({ linkGroupName, linkGroupImage, links }: QuickLinkGroupProps) => {
```

Comment:
I wonder if there's a way to combine the `QuickLink` and `QuickLinkGroup` components. Right now it's somewhat duplicated but we could possibly have an optional children param which we could use for setting the styling/adding the extra links around the icon.

What do you think?

Reply:
yeah I can combine them. One issue is that the Link wraps the single one, but not the group one, then that makes it so even the titles can't be shared.